eng/common/scripts/Verify-Links.ps1

```diff
@@ -134,6 +138,14 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 
 function CheckLink ([System.Uri]$linkUri)
 {
+  if(!$linkUri.ToString().Trim()) {
+    LogWarning "Found Empty link. Please use absolute link instead. Check here for more infomation: https://aka.ms/azsdk/guideline/links"
+    if ($checkedLinks.ContainsKey($emptyLinkMessage)) {
```

Comment:
Why do we need to special case this? I figured our relative link stuff would catch and handle this.

Reply:
I just want to store the empty case in checked link. However, it is not used anywhere. I will remove the check.